### ISSUE TITLE
Background colors do not sufficiently contrast text color

### ISSUE THREAD

**Issue by FireIsGood** [PamVanLonden/cognitive-styles#13]
Currently the `--orange` and `--blue` colors are used with white text on top. Checking the WCAG contrast ratio via [Adobe Color](https://color.adobe.com/create/color-contrast-analyzer) I noticed that blue and orange to the color white were **3.17 : 1** and **2.71 : 1** which are not compliant (>=4.5 for AA).

You may want to consider:

- Darker colors
- Using black on the colors
- Using a separate blue for backgrounds with white colors
- High contrast media query that does all of the above

**Comment by PamVanLonden** [comment]
Thank you. This is on my list to fix this summer. :-)
